webpack.config.js

```diff
@@ -50,7 +50,9 @@ module.exports = {
       },
     ],
   },
+  mode: 'development',
```

Comment:
This line can be removed. Our scripts will run the webpack base on mode, so this isn't needed